dpbench/benchmarks/kmeans/kmeans_numba_dpex_n.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 Intel Corp.
```

Comment:
For kmeans we only have kernel and prange implementations; no numpy implementation. kmeans_numba_dpex_n.py is the same as kmeans_numba_dpex_p.py. So we can remove kmeans_numba_dpex_n.py